### DISCUSSION TITLE
Hacktoberfest

### DISCUSSION THREAD

**Discussion by danibene** [neuropsychology/NeuroKit#717] [General]
Are you interested in participating? 

I've never done it before but it seems pretty simple & might bring new contributors: https://hacktoberfest.com/participation/ 
`Add the “hacktoberfest” topic to your repository to opt-in to Hacktoberfest and indicate you’re looking for contributions.`

**Comment by DominiqueMakowski** [comment]
Good idea! I just added the topic. We should probably think of flagging / creating some issues as "good for first contributions"
